### PR TITLE
Fetch, dispatch and style missions

### PR DESCRIPTION
**Pull request by ntazelo**
### Added features

**Style link when they are active**
**Fetch and dispatch mission when a user navigates on mission**
**Style mission page using react-bootstrap**

### CODE REVIEW

File: src/redux/actions/missions.js [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
-// API action creators
 const getMissions = () => async (dispatch) => {
   try {
-    const data = await api.fetchData(); // TODO: Change 'fetchData' to 'fetchMissions'
+    const data = await api.fetchMissions();
+    const filteredObject = data.map((obj) => {
+      const { mission_id: missionId, mission_name: missionName, description } = obj;
+      return { mission_id: missionId, mission_name: missionName, description };
+    });
+    console.log(filteredObject);
```
Comment (1GORDON) [inline src/redux/actions/missions.js:12]: # Optional Changes
 Please remember to remove the console.log in your missions.js.

**Review by 1GORDON — APPROVED** [review]
## STATUS: APPROVED 🟢 👏
Hello, @ntazelo great job working on the required changes 👍 💯

Now you can merge this branch 💪 😄
HAPPY CODING!! AND GOOD LUCK WITH THE NEXT ONES 🤞 🍀

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## STATUS: REQUIRED CHANGES ♻️ 
Hello, @ntazelo great job till now but it seems there's an error while running the code.

Make sure to fix it and again request code review.

**Review by sja-thedude — APPROVED** [review]
STATUS: APPROVED 🟢 👏
Hello, @ntazelo great job working on the required changes 👍 💯

Now you can merge this branch 💪 😄
HAPPY CODING!! AND GOOD LUCK WITH THE NEXT ONES 🤞 🍀

**Pull request merged**